code/workspaces/web-app/src/components/stacks/StackCard.js

```diff
@@ -212,7 +212,7 @@ function generateGetImage(classes, typeName) {
     const initial = stackDescription && stackDescription.initial && getDisplayName(stack).charAt(0);
 
     if (logoImage) {
-      return <img className={classes.cardImage} src={logoImage} alt={stackDescription} />;
+      return <img className={classes.cardImage} src={logoImage} alt={stackDescription.description} />;
```

Comment:
Good spot that this was using the wrong thing! However, I don't think this is the right way to be using the image `alt` label as it should be used to describe the image. This description should likely be surfaced in the create notebook form instead. Maybe this should be added as a technical debt ticket or similar?

Reply:
Thanks Alex.  I've updated NERCDL-737 (Create design document for new form design) with a note that we're currently not using this description.
For the image alt, I could change it to
```
alt={`${stack.type} logo`}
```
so that it describes the image

Reply:
Brilliant, thanks for updating the ticket with the extra information. 
I think the suggested alt label would be a good starting point and would be good for merging this PR. Interestingly, I think if an image isn't important to the understanding of the page, then it can be given an empty alt tag and screen readers etc. will ignore it. For now I think it is important as it is the only way of knowing what type of stack it is, but this is something that might change with some of the accessibility tickets I created the other day. I will update one of those tickets with a note to research how best to specify the type of something in a list like this e.g. Jupyter.

Reply:
I've updated NERCDL-806 with this information.